packages/mdc-linear-progress/_mixins.scss

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/theme/functions";
```

Comment:
mdc-theme-prop-value is defined in _variables.scss instead of _functions.scss...so should theme/variables be imported instead?
Related, maybe we can move mdc-theme-prop-value back into _functions.scss because it doesn't look like mdc-theme-prop-value is used anywhere else (double check in case I'm missing something)

Reply:
Done! Good catch.

I'm also using `mdc-theme-prop-value` in #1544.

The real problem, though, is that `mdc-theme-prop-value` needs to be able to read `$mdc-theme-property-values`, which is defined in `_variables.scss`.

Unfortunately, `_variables.scss` `@import`s `_functions.scss`, so if we moved `mdc-theme-prop-value` to `_functions.scss`, we would be creating a circular reference. I actually tried doing that originally but the compiler blew up 😆